log_outgoing_requests/management/commands/cleanup_requests_logs.py

```diff
@@ -0,0 +1,8 @@
+from django.core.management.base import BaseCommand
+
+from log_outgoing_requests.models import OutgoingRequestsLog
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        OutgoingRequestsLog.objects.cleanup_requests_logs()
```

Comment:
A bit of output/feedback would be appreciated. E.g. "Deleted {num} log record(s)."

Reply:
That's logged as part of the `cleanup`/`prune` method already.